environmental_agency/demo/account_move_line.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="account_move_line_19" model="account.move.line">
+        <field name="partner_id" ref="res_partner_37"/>
+        <field name="product_id" ref="product_product_8"/>
+        <field name="move_id" ref="account_move_8"/>
+        <field name="esg_emission_factor_id" ref="esg_emission_factor_1066"/>
+        <field name="balance">75.0</field>
+        <field name="price_unit">15.0</field>
+        <field name="display_type">product</field>
+        <field name="quantity">5.0</field>
+        <field name="name">Office Chairs</field>
+        <field name="debit">75.0</field>
+        <field name="amount_currency">75.0</field>
```

Comment:
```suggestion
        
```

Reply:
The amount expressed in an optional other currency if it is a multi-currency entry.